user-guide/Advanced_Functionality/DataMiner_Systems/BPA_tests/BPA_Check_Agent_Presence.md

```diff
@@ -0,0 +1,54 @@
+---
+uid: BPA_Check_Agent_Presence
+---
+
+# Check Agent Presence Test In NATS
+
+NATS serves as an inter-DMA message broker. This means that every DataMiner Agent must be able to reach every other DataMiner Agent in the cluster over NATS.
+
+This BPA test will scan the NATS info as seen by this Agent. It will verify if it finds a route to every Agent it is aware of, and it will perform a NATS client connection test to each Agent.
+
+Depending on the configuration of the system, the NATS info is gathered from SLNet (via SLCloud.xml) or via BrokerGateway. An extra check is performed for cluster consistency if DataMiner uses BrokerGateway.
+
+This BPA test is available on demand. You can [run it in System Center](xref:Running_BPA_tests) (on the *Agents > BPA* tab). From DataMiner 10.3.12 onwards, it is available by default.
```

Comment:
Which release note added this BPA? I found RN [37190 ](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=37190)for this, but it does not have any release versions filled in.

Reply:
It looks like RN37190 was abandoned and can probably be removed.

Reply:
I believe this [RN36837](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=36837) has added the BPA. however, I see that no versions were filled in here as well.
From [RN38361](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=38361) on, BrokerGateway setups are compatible with the BPA and the extra check for cluster consistency is then possible.

I am wondering if this should be added, from which version on this is possible? As brokerGateway isn't fully released yet.

Reply:
I've checked it manually actually. It wasn't in the released package of 10.3.11 but it was in the package with version 10.3.12. 
As this wasn't documented correct, I think this might be the only option to know when it was added?